internal/ir/infra.go

```diff
@@ -209,5 +210,5 @@ func (p *ProxyInfra) ObjectName() string {
 	if len(p.Name) == 0 {
```

Comment:
I dont think `ObjectName` is called by anyone, can we delete it ?
cc @danehans

Reply:
Yeah I started here but didn't see it called anywhere so I wasn't sure.

Reply:
Removed the change here since it wasn't needed